### PR TITLE
FXIOS-1885: remove leanplum repowide

### PR DESCRIPTION
**Pull request by nishant2718**
# Overview
This ticket addresses this [JIRA ticket](https://jira.mozilla.com/browse/FXIOS-1885), and #8579. This work is related to PR https://github.com/mozilla-mobile/firefox-ios/pull/8561.

### CODE REVIEW

**Comment by nishant2718** [comment]
Should we remove all buddyBuild related artifacts as well, as part of this PR?

**Comment by nbhasin2** [comment]
> Should we remove all buddyBuild related artifacts as well, as part of this PR?

Lets make another PR for BB just to keep things clean. We call those PR [nobug, build] PRs :)

**Review by nbhasin2 — APPROVED** [review]

**Comment by nbhasin2** [comment]
@nishant2718 I checked your BR scripts and they look good. Just wait for builds to be green before you merge.

**Pull request merged**
